packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -97,6 +99,11 @@ interface IUnreferencedEvent {
     viaHandle?: boolean;
 }
 
+interface IGCReference {
```

Comment:
We already have a node -> outboundRoutes structure. Adding another for each node -> route combination doesn't add any value.